.github/workflows/linting.yml

```
@@ -66,6 +66,22 @@ jobs:
       - name: TFLint Modules
         run: cd modules && tflint --recursive -f compact
 
+      - name: TFLint FAST
+        run: |
+          # TODO: try to generalize this
+          tflint --chdir fast/stages/0-bootstrap --var-file ../../../tests/fast/stages/s0_bootstrap/simple.tfvars
+          tflint --chdir fast/stages/1-resman --var-file ../../../tests/fast/stages/s1_resman/simple.tfvars
+          tflint --chdir fast/stages/2-networking-a-peering --var-file ../../../tests/fast/stages/s2_networking_a_peering/common.tfvars
+          tflint --chdir fast/stages/2-networking-b-vpn --var-file ../../../tests/fast/stages/s2_networking_b_vpn/common.tfvars
+          tflint --chdir fast/stages/2-networking-c-nva --var-file ../../../tests/fast/stages/s2_networking_c_nva/common.tfvars
+          tflint --chdir fast/stages/2-networking-d-separate-envs --var-file ../../../tests/fast/stages/s2_networking_d_separate_envs/common.tfvars
+          tflint --chdir fast/stages/2-security --var-file ../../../tests/fast/stages/s2_security/common.tfvars
+          tflint --chdir fast/stages/3-data-platform/dev --var-file ../../../../tests/fast/stages/s3_data_platform/common.tfvars
+          tflint --chdir fast/stages/3-gcve/prod/ --var-file ../../../../tests/fast/stages/s3_gcve_minimal/simple.tfvars
+          tflint --chdir fast/stages/3-data-platform/dev --var-file ../../../../tests/fast/stages/s3_data_platform/common.tfvars
+          tflint --chdir fast/stages/3-gke-multitenant/dev --var-file ../../../../tests/fast/stages/s3_gke_multitenant/common.tfvars
+          tflint --chdir fast/stages/3-project-factory/dev --var-file ../../../../tests/fast/stages/s3_project_factory/common.tfvars
+
```

```suggestion
          for stage in 0-bootstrap 1-resman ; do
              tflint --chdir fast/stages/${stage} --var-file ../../../tests/fast/stages/s${${stage}/simple.tfvars
          done

          for stage in 2-networking-a-peering 2-networking-b-vpn 2-networking-c-nva 2-networking-d-separate-envs 2-security  ; do
              tflint --chdir fast/stages/${stage} --var-file ../../../tests/fast/stages/s${${stage}/common.tfvars
          done

          for stage in  3-data-platform  3-gke-multitenant 3-project-factory ; do
              tflint --chdir fast/stages/${stage}/dev --var-file ../../../tests/fast/stages/s${${stage}/common.tfvars
          done

          tflint --chdir fast/stages/3-gcve/prod/ --var-file ../../../../tests/fast/stages/s3_gcve_minimal/simple.tfvars
```

Something like that? We could rename `simple.tfvars` to `common.tfvars` to simplify more, and change the directory for `3-gcve/prod` to `3-gcve/dev`.

---

@wiktorn I'll resolve this. Feel free to open a new PR to implement your suggestions (rename s3_gcve_minimal -> s3_gcve, 3-gcve/prod -> 3-gcve/dev, and always use common.tfvars)

cc @eliamaldini

---

```suggestion
for stage_dir in $(find fast/stages -name main.tf -printf '%h\n' ) ; do
  STAGE_NAME=$(echo ${stage_dir} | sed -e 's#^fast/stages/\([^/]*\).*$#\1#')
  tflint --chdir ${stage_dir} --var-file ../../tests/${stage_dir}/common.tfvars
done
```
+ couple renames?